logstage-elastic/src/main/scala/Logging.scala

```diff
@@ -17,8 +17,15 @@ trait Logging extends LoggerTags {
 
   implicit lazy val log: IzLogger = {
     val legacyNameNormalisation = sys.props.get("logger.izumi.legacyNameNormalisation").contains("true")
-
-    lazy val jsonPolicy = LogstageCirceElasticRenderingPolicy(loggerClass, legacyNameNormalisation)
+    // set this to 0 to turn it off
+    val truncateStringValues = sys.props.get("logger.izumi.truncateStringValuesTo").map(_.toInt)
```

Comment:
Is it supposed to fail when it cannot be parsed to Long?

Reply:
yep. I reckon better to fail loud at invalid system properties